app/controllers/posts_controller.rb

```diff
@@ -684,8 +684,16 @@ def create_params
       :draft_key
     ]
 
-    Post.plugin_permitted_create_params.each do |key, plugin|
-      permitted << key if plugin.enabled?
+    Post.plugin_permitted_create_params.each do |key, value|
```

Comment:
Could you add some tests to ensure this API does not regress?

Reply:
@eviltrout Tests have been added